index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+        <section class="navbar">
+            <nav>
+                <a href="#">John Doe</a>
+                <a href="#"><img src="./Microverse-Student-Project-toolbar/Tertiary/Enabled.png" alt="menu icon"></a>
+            </nav>
+        </section>
+
+    <section class="headline">
+        <h1 class="primary-text">Hey There.<br>I'm Alex</h1>
+        <h3 class="supporting-text">I'am a Software Developer</h3>
+        <p class="description"> I can help you build a product , feature or
+            website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don't hestiate to contact me.</p>
+            
+        <div class="social-media">
+            <img src="./Microverse-Student-Project/Tertiary/Disabled.png" alt="Github icon">
+            <img src="./Microverse-Student-Project-1(1)/Tertiary/Disabled.png" alt="Linkedin icon">
+            <img src="./Microverse-Student-Project-1(2)/Tertiary/Enabled.png" alt="icon">
+            <img src="./Microverse-Student-Project-1(3)/Tertiary/Disabled.png" alt="Twitter icon">
+            <img src="./Microverse-Student-Project-1(4)/Tertiary/Disabled.png" alt="Medium icon">   
+        </div>    
```

Comment:
- [ OPTIONAL ] In this section the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Bellow :point_down: :point_down:  you can see a good implementation of this part.
- This is optional because it was not pointed out by the previous reviewer :upside_down_face::upside_down_face:
```
<ul class="social-media">
    <li>
      <a href="https://github.com/githubhandle" class="icons">
        <img src="./Microverse-Student-Project/Tertiary/Disabled.png" alt="Github icon" />
      </a>
   </li>
    ....
 </ul>
```